ambry-api/src/main/java/com.github.ambry/network/NetworkSend.java

```diff
@@ -59,7 +51,7 @@ public Send getPayload() {
 
   public void onSendComplete() {
     if (metrics != null) {
-      metrics.updateResponseSendTime(SystemTime.getInstance().milliseconds() - sendStartTimeInMs);
+      metrics.updateSendTime(time.milliseconds() - sendStartTimeInMs);
```

Comment:
Yes, its the time from which the NetworkSend was queued up until it was dequeued to be sent via selector. From client's perspective, this is time from which the networkClient receives a new send from the caller of sendAndPoll(). For Server, it is the time at which the "Send" was sent to RequestResponseChannel(i.e. SocketRequestResponseChannel).